### DISCUSSION TITLE
READMEs not displaying

### DISCUSSION THREAD

**Discussion by ctjlewis** [npm/feedback#904] [Registry]
- https://twitter.com/ctjlewis/status/1646254813342990343
- https://www.npmjs.com/package/openai-streams

Seems to be an issue that started today or within last few days. README displays on GitHub and is confirmed in `npm pack` tarball, NPM still says "no README".

**Comment by davestewart** [comment]
I came here to report the same issue after emailing support.

They suggested I come here and flag it too.

Any time from 30 - 90 minutes last week, which is not great for OSS projects!

**Comment by MikeMcC399** [comment]
This is happening again. In the section "Recently updated packages" on https://www.npmjs.com/ none of the packages which have been published today are showing README files.

For example https://www.npmjs.com/package/@thirdweb-dev/react 3.14.39 • Public • Published 3 hours ago shows

https://www.npmjs.com/package/@thirdweb-dev/react?activeTab=readme
**This package does not have a README.**

whereas
https://www.npmjs.com/package/@thirdweb-dev/react?activeTab=code clearly lists a
`README.md`

file.

**Reply by MikeMcC399** [reply]
The problem with the display of READMEs is no longer occurring.
